README.md

```diff
@@ -375,7 +375,7 @@ print(balance)
 
 ## Create transfers
 
-You can also create transfers in the SDK (TED/Pix).
+You can also create transfers in the SDK (TED/Pix), and configure transfers behavior according to its rules.
```

Comment:
Aqui não precisa da vírgula depois do and. Só coloca vírgula quando forem frases independentes.

Reply:
tranfers -> transfer